pkg/motor/interface.go

```diff
@@ -25,7 +24,7 @@ type MotorNode interface {
 	GetPubKey() *secp256k1.PubKey
 	GetDID() did.DID
 	GetDIDDocument() did.Document
-	GetHost() host.SonrHost
+	// GetHost() host.SonrHost
```

Comment:
This doesnt need to be commented out because of my PR to isolate the libp2p connection process

Reply:
On android you would simply not call `Connect()`